patches/tModLoader/Terraria/ModLoader/UI/UIModSources.cs

```diff
@@ -412,9 +412,8 @@ private static bool DoesDotnetWork(string path)
 	internal void Populate()
 	{
 		Task.Run(() => {
-			var modSources = ModCompile.FindModSources();
+			var modSources = ModCompile.FindModSources(refindAllMods: true, out var modFiles);
```

Comment:
Lets keep it simple. 
```cs
// call before finding mod sources, in case someone has built a new mod externally after opening the game but before re-opening the sources menu
var modFiles = ModOrganizer.FindAllMods(); 
var modSources = ModCompile.FindModSources();
```